server/src/main/java/com/objectcomputing/checkins/services/member_skill/skillsreport/SkillsReportController.java

```diff
@@ -0,0 +1,55 @@
+package com.objectcomputing.checkins.services.member_skill.skillsreport;
+
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.MediaType;
+import io.micronaut.http.annotation.*;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.micronaut.scheduling.TaskExecutors;
+import io.netty.channel.EventLoopGroup;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.inject.Named;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.concurrent.ExecutorService;
+
+@Controller("/reports/skills")
+@Secured(SecurityRule.IS_AUTHENTICATED)
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "skills-report")
```

Comment:
Should be /reports/skills, per issue #675 (bringing comment over from other PR)

Reply:
sorry, I was miss reading this...that's what I get for rushing.

Reply:
So is the tag name `skills-report` fine? or should it also be `/reports/skills`?

Reply:
it's fine. we need to reorganize all of them anyway later.